bayarea/datasources.py

```diff
@@ -8,106 +8,115 @@
+@orca.table('buildings', cache=True)
+def buildings(store):
+    df = store['buildings']
+    return df
+
+@orca.table('building_types', cache=True)
+def buildings(store):
+    df = store['building_types']
+    return df
+
+@orca.table('jobs', cache=True)
+def jobs(store):
+    df = store['jobs']
     return df
 
 
-############################################################
+@orca.table('establishments', cache=True)
+def establishments(store):
+    df = store['establishments']
+    return df
 
-# Table of Rental Data from  Craigslist, bayarea_urbansim added by Arezoo
 
-@orca.table(cache=True)
-def craigslist():
-    df = pd.read_csv(
-        d + 'MTC_craigslist_listings_7-10-18.csv',
-        index_col='pid', dtype={'pid': int})
+@orca.table('households', cache=True)
+def households(store):
+    df = store['households']
     return df
 
 
-@orca.table(cache=True)
-def rentals():
-    df = pd.read_csv(
-        d + 'rentals_with_nodes.csv',
-        index_col='pid', dtype={'pid': int, 'rent': float})
+@orca.table('persons', cache=True)
+def persons(store):
+    df = store['persons']
     return df
 
 
-############################################################
+@orca.table('rentals', cache=True)
+def rentals(store):
+    rentals = store['rentals']
+    rentals.rent[rentals.rent < 100] = 100
+    rentals.rent[rentals.rent > 10000] = 10000
+
+    rentals.rent_sqft[rentals.rent_sqft < .2] = .2
+    rentals.rent_sqft[rentals.rent_sqft > 50] = 50
+    return rentals
+
+
+@orca.table('units', cache=True)
+def units(store):
+    df = store['units']
+    df.index.name = 'unit_id'
+    return df
 
-# Tables synthesized by Max Gardner
 
-@orca.table(cache=True)
-def units():
-    df = pd.read_csv(
-        d + 'units_v2.csv',
-        index_col='unit_id', dtype={'unit_id': int, 'building_id': int})
+@orca.table('nodessmall', cache=True)
+def nodessmall(store):
+    df = store['nodessmall']
+    df.index.name = 'osmid'
     return df
 
 
-@orca.table(cache=True)
-def households():
-    df = pd.read_csv(
-        d + 'households_v2.csv',
-        index_col='household_id', dtype={
-            'household_id': int, 'block_group_id': str, 'state': str,
-            'county': str, 'tract': str, 'block_group': str,
-            'building_id': int, 'unit_id': int, 'persons': float})
+@orca.table('edgessmall', cache=True)
+def edgessmall(store):
+    df = store['edgessmall']
     return df
 
 
-@orca.table(cache=True)
-def persons():
-    df = pd.read_csv(
-        d + 'persons_v3.csv',
-        index_col='person_id', dtype={'person_id': int, 'household_id': int})
+@orca.table('nodeswalk', cache=True)
+def nodeswalk(store):
+    df = store['nodeswalk']
+    df.index.name = 'node_id_walk'
     return df
 
 
-@orca.table(cache=True)
-def jobs():
-    df = pd.read_csv(
-        d + 'jobs_v2.csv',
-        index_col='job_id', dtype={'job_id': int, 'building_id': int})
+@orca.table('edgeswalk', cache=True)
+def edgessmall(store):
```

Comment:
@waddell I think this should read `def edgeswalk(store)`, right?

Reply:
Yup. That's a typo. Fixed.